src/ngraph/descriptor/layout/tensor_view_layout.hpp

```diff
@@ -50,7 +50,7 @@ namespace ngraph
                 ///
                 /// When we support non-linear buffers, this will need to be something other than size_t.
                 virtual size_t get_size() = 0;
-
+                virtual size_t size();
```

Comment:
In STL, `size()` is element size, not byte size, so change this to `get_byte_size()` or `get_allocated_size()`.